charts/openebs/templates/localprovisioner/device-class.yaml

```diff
@@ -1,7 +1,9 @@
 {{- if .Values.localprovisioner.enabled }}
 {{- $localpvprovisionerValues := index .Values "localpv-provisioner" }}
 {{- if not $localpvprovisionerValues.enabled }}
-{{- if .Values.localprovisioner.enableDeviceClass }}
+# The second operand in the OR operation can be remove
+# when enableDeviceClass is deprecated.
+{{- if and (eq .Values.localprovisioner.deviceClass.enabled true) (eq .Values.localprovisioner.enableDeviceClass true) }}
```

Comment:
Shouldn't this be `or` instead of `and`

Reply:
It should be -- FALSE or FALSE, that would make more intuitive sense...
But I made it -- TRUE and TRUE

Both of these will be set to true by default, we want to be able to stop the resource creation if either of the two is set to false.

Can you help me with the syntax please? not sure how to write a FALSE or FALSE correctly...

Reply:
> Both of these will be set to true by default, we want to be able to stop the resource creation if either of the two is set to false.

if thats the behaviour we want, then the current logical operation seems good to me, 

Just one small change, since `.Values.localprovisioner.<class>.enabled` is a boolean, you can simplify and skip the equal check
```
{{- if and .Values.localprovisioner.deviceClass.enabled .Values.localprovisioner.enableDeviceClass }}
```